app/views/feedback/_outlined_flash_message.html.haml

```diff
@@ -2,4 +2,13 @@
   .govuk-body.mv0{ class: "govuk-#{style}-summary__body" }
     %a.govuk-link.govuk-link--no-visited-state.govuk-success-summary__dismiss-link.js-dismissable__link{ href: '#' }
       = t('buttons.dismiss')
+    - if defined?(heading)
```

Comment:
I would just check the action here, you already have `@vacancy` defined and accessible, so no need to pass heading and links local variables

Reply:
Heading and links are passed because they might not always be relevant when using this partial